### ISSUE TITLE
no cycle should not trigger inside react redux actions

### ISSUE THREAD

**Issue by fttriquet** [benmosher/eslint-plugin-import#1463]
**Repro**

```
// cycle/action.js

import setLoading from '../loading/actions';

export function doSomething() {
  return (dispatch) => {
    dispatch(setLoading(true));
  }
}

export function makeACycle() {
...
}
```
// loading/action.js

import makeACycle from '../cycle/actions';

export function setError() {
  return (dispatch) => {
    dispatch(makeACycle(true));
  }
}
```

**Actual Behavior**

Trigger no-cycle warnings, even though actions are not calling themselves.

**Expected Behavior**

Action files should be ignored ?
